index.js

```diff
@@ -85,12 +102,12 @@ Block.prototype.isGenesis = function () {
  * @method setGenesisParams
  */
 Block.prototype.setGenesisParams = function () {
```

Comment:
Should we move this code into a new BlockHeader.setGenesisParams() method? That way it will be consistent with how .hash() and .isGenesis() work (both forward the calls to BlockHeader)